client/src/pages/admin/reservations/ReservationCard.tsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import {
+  Alert,
+  Avatar,
+  Button,
+  Card,
+  CardMedia,
+  Stack,
+  Typography,
+} from "@mui/material";
+import ChatBubbleOutlineIcon from "@mui/icons-material/ChatBubbleOutline";
+import ReservationAttachment from "./ReservationAttachment";
+import Reservation from "../../../types/Reservation";
+
+interface ReservationCardProps {
+  reservation: Reservation;
+}
+
+const weekdays = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+];
+
+const months = [
+  "Jan.",
+  "Feb.",
+  "March",
+  "April",
+  "May",
+  "June",
+  "July",
+  "Aug.",
+  "Sept.",
+  "Oct.",
+  "Nov.",
+  "Dec.",
+];
+
+function formatDate(dateTime: string) {
+  const d = new Date(dateTime);
+  return `${weekdays[d.getDay()]}, ${months[d.getMonth()]} ${d.getDate()}`;
+}
+
+function formatTime(dateTime: string) {
+  const d = new Date(dateTime);
+  return d.toLocaleTimeString().replaceAll(":00", "");
```

Comment:
If a time was 7:00:02 would this just read as 7:02?

Reply:
Yeah good point. I'll fix it and ping you when its ready.

Reply:
@Axj2689 I updated the time formatting, pls take another look <3